manager/cve_handler.py

```diff
@@ -39,7 +39,12 @@ class SystemCvesView(ListView):
     """Database select for /cves/{cve_id}/affected_systems API endpoints"""
 
     def __init__(self, rh_account_id, synopsis, cves_without_errata, list_args, parsed_args, uri, ids_only=False):
-        query = self._full_query(rh_account_id, synopsis) if not ids_only else self._id_query(rh_account_id, synopsis)
+        if not cves_without_errata:
```

Comment:
there will need to be different logic, `cves_without_errata` is a feature flag deciding if the functionaly is available at all

then it'll depend on `advisory_available` filter value if only the original full query is returned, only the unfixed query is returned or data from both combined